_posts/newsletters/2021-03-22-newsletter.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: US-RSE Newsletter
+subtitle: March 2021 Newsletter
+category: newsletter
+tags: [newsletter, DEI, working groups, governance, career panel, geosciences]
+---
+
+In this bi-monthly newsletter, we share recent, current, and planned activities of the US-RSE Association, and related news that we think is of interest to US-RSE members. Newsletters are also available on our [website](https://us-rse.org/newsletters/) alongside the growing resources and information on the US-RSE Association. A sign-up option for our newsletter is available [here](https://us-rse.org/join/).
+
+In this issue:
+
+* [Updates on Working Groups](#updates)
+* [Inaugural Event of the DEI Speaker Series](#deispeaker)
+* [US-RSE Finances and Governance Proposal](#governance)
+* [US-RSE Community Calls](#communitycall)
+* [International Council of RSE Associations](#council)
+* [ADSA/US-RSE Career Panel](#panel)
+* [Geosciences Data & Software Help Desk Opportunity](#geohelp)
+* [Interesting Events](#events)
+* [Recent Job Postings](#jobs)
+* [Interesting Reads](#reads)
+* [US-RSE Outreach](#outreach)
+* [Get Involved](#involved)
+
+<a name="wg"></a>
+# **Updates on Working Groups**
+
+**Diversity, Equity and Inclusion (DEI):** The US-RSE DEI Working Group is working to bring the US-RSE community quarterly visits from RSE professionals who have vested interests or who research DEI in the RSE community at large. The inaugural event will take place Thursday March 25, 2021, with Cherri Pancake as speaker (see below for details). Furthermore, we have worked on a mission statement regarding DEI for US-RSE and will open it up soon for comments and publish it afterwards on the website. If you’d like to discuss and/or contribute, please contact us on the *[#dei-discussion](https://usrse.slack.com/archives/C01C8CJQ7AP)* Slack channel or contact Lance Parsons ([lparsons@princeton.edu](mailto:lparsons@princeton.edu)) and we’ll help you get involved.
```

Comment:
```suggestion
**Diversity, Equity and Inclusion (DEI):** The US-RSE DEI Working Group is working to bring the US-RSE community quarterly visits from professionals who have vested interests or who research DEI in the community at large. The inaugural event will take place Thursday March 25, 2021, with Cherri Pancake as speaker (see below for details). Furthermore, we have worked on a mission statement regarding DEI for US-RSE and will open it up soon for comments and publish it afterwards on the website. If you’d like to discuss and/or contribute, please contact us on the *[#dei-discussion](https://usrse.slack.com/archives/C01C8CJQ7AP)* Slack channel or contact Lance Parsons ([lparsons@princeton.edu](mailto:lparsons@princeton.edu)) and we’ll help you get involved.
```
I don't think the professionals / their research has to be RSE specific. I think it would be okay to just remove those qualifiers. There very likely are professionals in the more broad tech / company landscape that we would invite to talk that don't go under the umbrella of RSE.

Reply:
@vsoch - I agree, but am a bit hesitant to change it in case it changes the intent of the DEI group. I haven't been part of the discussion recently. 
@lparsons - this ok?

Reply:
I would agree that we remove the qualifiers. Thanks!

Reply:
I agree too